HelpSource/Classes/Server.schelp

```diff
@@ -623,6 +623,13 @@ an optional instance of link::Classes/Condition:: used for evaluating this.
 argument:: ... args
 one or more valid OSC messages.
 
+method:: isLocal
+Returns true if current server is local
```

Comment:
Doesn't add much. I suggest "Returns true if the server is local, i.e. running on the same machine as the language. `s.isLocal` is equivalent to `s.addr.isLocal`."